[Github][Bugfix] Auto PR Creation

# Description

Git bot can not longer auto create pull requests, check a way to fix this

`pull request create failed: GraphQL: GitHub Actions is not permitted to create or approve pull requests (createPullRequest)
Error: Process completed with exit code 1.`

--- a/.github/workflows/auto-create-pr.yml
+++ b/.github/workflows/auto-create-pr.yml
@@ -3,31 +3,11 @@ name: Auto Create PR from Issue
 on:
   issue_comment:
     types: [created]
-  workflow_dispatch:
-    inputs:
-      issue_number:
-        description: "Issue number for testing"
-        required: true
-      issue_title:
-        description: "Issue title"
-        required: true
-      comment_body:
-        description: "Comment text (e.g. /create-pr /feature)"
-        required: true
 
 jobs:
   create-branch-and-pr:
-    if: >
-      (github.event_name == 'workflow_dispatch') ||
-      (
-        github.event.issue.pull_request == null &&
-        contains(github.event.comment.body, '/create-pr') &&
-        (
-          github.event.comment.author_association == 'MEMBER' ||
-          github.event.comment.author_association == 'OWNER'
-        )
-      )
-
+    # Only run if comment is on an issue (not PR) and contains the trigger.
+    if: github.event.issue.pull_request == null && contains(github.event.comment.body, '/create-pr')
     runs-on: ubuntu-latest
 
     permissions:
@@ -66,7 +46,7 @@ jobs:
           # Clean up the title for the branch name.
           CLEAN_TITLE=$(echo "$ISSUE_TITLE" | tr '[:upper:]' '[:lower:]' | tr -cs '[:alnum:]' '-' | sed 's/^-//; s/-$//' | cut -c1-50)
           BRANCH_NAME="${BRANCH_TYPE}/${CLEAN_TITLE}"
-
+          
           echo "branch_name=$BRANCH_NAME" >> $GITHUB_OUTPUT
           echo "branch_type=$BRANCH_TYPE" >> $GITHUB_OUTPUT
           echo "issue_number=$ISSUE_NUMBER" >> $GITHUB_OUTPUT
@@ -86,12 +66,11 @@ jobs:
           GH_TOKEN: ${{ secrets.GH_PAT }}
         run: |
           # Use the issue author
-          if [ "${{ github.event_name }}" = "workflow_dispatch" ]; then
-            ASSIGNEE="${{ github.actor }}"
-          else
-            ASSIGNEE="${{ github.event.issue.user.login }}"
+          if [ -z "$ASSIGNEES" ]; then
+            ASSIGNEES="${{ github.event.issue.user.login }}"
           fi
-          echo "assignees=$ASSIGNEE" >> $GITHUB_OUTPUT
+          
+          echo "assignees=$ASSIGNEES" >> $GITHUB_OUTPUT
 
       - name: Create Draft PR
         env:
@@ -109,7 +88,7 @@ jobs:
           
           # Create draft PR with template body and assignees
           gh pr create \
-            --title "${{ github.event_name == 'workflow_dispatch' && github.event.inputs.issue_title || github.event.issue.title }}" \
+            --title "${{ github.event.issue.title }}" \
             --body "$PR_BODY" \
             --draft \
             --base ${{ github.event.repository.default_branch }} \

